examples/backends/vllm/launch/agg_omni_cosmos3_image.sh

```
@@ -0,0 +1,60 @@
+#
+# Aggregated Cosmos3 text-to-image generation (1 GPU).
+# Uses the native vLLM-Omni Cosmos3 pipeline; --no-cosmos3-guardrails skips
+# loading the safety guardrail models. A worker serves a single modality, so
+# this script registers the "image" modality (see agg_omni_cosmos3_video.sh
+# for text-to-video).
+
+set -e
+trap 'echo Cleaning up...; kill 0' EXIT
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPT_DIR/../../../common/launch_utils.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../../../common/launch_utils.sh was not specified as input (see shellcheck -x).

(SC1091)